flexget/plugins/sites/argenteam.py

```diff
@@ -0,0 +1,90 @@
+from __future__ import unicode_literals, division, absolute_import
+from builtins import *  # noqa pylint: disable=unused-import, redefined-builtin
+
+import logging
+
+from flexget import plugin
+from flexget.entry import Entry
+from flexget.event import event
+from flexget.config_schema import one_or_more
+from flexget.utils.requests import Session, TimedLimiter, RequestException
+from flexget.utils.search import normalize_scene
+from flexget.plugin import PluginError
+
+log = logging.getLogger('argenteam')
+
+requests = Session()
```

Comment:
Shouldn't use this. Use `task.requests` in the `search` function instead.

Reply:
Sorry. I ralied on the rarbg plugin...